src/Murder.Editor/CustomEditors/WorldAssetEditor.cs

```diff
@@ -488,7 +488,7 @@ private void MoveGroupOfEntities(Guid roomEntity, Point from, Point to, string g
         {
             GameLogger.Verify(_world is not null);
 
-            Point worldDelta = (to - from) * Grid.CellSize;
+            Point worldDelta = (to - from) * Game.Instance.GridConfiguration.CellSize;
```

Comment:
can we shortcut to `Game.Grid`? (we use this everywhere so I'm okay enough losing clarity)

Reply:
....I'm okay going further and doing a static class for `Grid` that access the `Game` field